packages/tasks/src/model-libraries.ts

```diff
@@ -166,6 +166,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"adapter_config.json"`,
 	},
+	cosmos: {
```

Comment:
Unrelated to this PR but are these models supposed to be usable with Nemo ? Since they are tagged as "nemo", there is a "use this model" snippet in all of them but with empty snippets. This is due to the fact that the nemo code snippet generator is incomplete (see here https://github.com/huggingface/huggingface.js/blob/504fcb4fd8d9774327770a3b949d9812823138e0/packages/tasks/src/model-libraries-snippets.ts#L1082 and https://github.com/huggingface/huggingface.js/blob/504fcb4fd8d9774327770a3b949d9812823138e0/packages/tasks/src/model-libraries-snippets.ts#L959). @Haoxiang-Wang @pcuenca  would it make sense to open a separate PR to complete the snippets?